frontend/src/components/TransactionItem.tsx

```diff
@@ -74,14 +74,22 @@ function TransactionItem({ tx }: Props) {
   const pubkey = tx.metadata?.nostr?.pubkey;
   const npub = pubkey ? safeNpubEncode(pubkey) : undefined;
 
-  const from = tx.metadata?.payer_data?.name
-    ? `from ${tx.metadata.payer_data.name}`
+  const payerName = tx.metadata?.payer_data?.name;
+  const from = payerName
+    ? `from ${payerName}`
     : npub
       ? `zap from ${npub.substring(0, 12)}...`
       : undefined;
 
+  const recipientIdentifier = tx.metadata?.recipient_data?.identifier;
+  const to = recipientIdentifier
+    ? `${tx.state === "failed" ? "sending " : ""}to ${recipientIdentifier}`
```

Comment:
This looks wrong. What's it supposed to say? if it failed it doesn't mean "sending"

Reply:
"Failed to send to" didn't feel good because of two tos so I changed it to be "Failed sending to" instead

Reply:
Ah, I see "Failed " is in front. Maybe "Failed payment" rather than "Failed sending"?

Reply:
Sounds better, will change 🙌